### PR TITLE
PREAPPS-4971: Fix reminders refetch only when notifications for appt present

### PR DESCRIPTION
**Pull request by yashrajpchavda**
- updated noop code to include parameters

### CODE REVIEW

File: src/batch-client/index.ts [hunk 1]
```diff
@@ -1704,7 +1706,10 @@ export class ZimbraBatchClient {
 			this.sessionId ||
 			(this.sessionHandler && this.sessionHandler.readSessionId()),
 		origin: this.origin,
-		userAgent: this.userAgent
+		userAgent: this.userAgent,
+		...(this.notifier.getSequenceNumber() && {
+			sessionSeq: this.notifier.getSequenceNumber()
```
Comment (silentsakky) [inline src/batch-client/index.ts:1711]: are we maintaining same seq number when request is fired through old zimbra-client?

File: src/request/index.ts [hunk 2]
```diff
@@ -247,14 +247,23 @@ export function jsonRequest(
 		[soapRequestName]: soapCommandBody(options)
 	};
 
+	let fetchOptions;
+
+	if (requestOptions.fetchOptions) {
+		fetchOptions = {
+			signal: requestOptions.fetchOptions.signal
```
Comment (silentsakky) [inline src/request/index.ts:254]: what is signal?

Comment (silentsakky) [reply src/request/index.ts:254]: can't we just send it as requestOptions.signal? instead of adding fetchOptions layer?

Comment (silentsakky) [reply src/request/index.ts:254]: its for abortcontroller, so ignore above comments

File: src/batch-client/index.ts (hunk 1 above)
Comment (yashrajpchavda) [reply src/batch-client/index.ts:1750]: Ah, had missed that. The answer is, no we are not. I just checked that only `csrf` is added in old client, it is not adding `userAgent`, `sessionId`, `sessionSeq`, `accountName`, and `jwt`. Should we add them too?

**Pull request merged**